### ISSUE TITLE
[FIXED] Mission Control does not bind to any key, as a program / fn key / or macro

### ISSUE THREAD

**Issue by NathanJava** [ckb-next/ckb-next#486]
Hello, I'm trying to rebuild CKB after Catalina destroyed it, and everything is going well except I can no longer bind the mission control app to my Sniper key on Harpoon RGB..

I tried to bind it to a program (open '/Applications/Mission Control.app') but that no longer works. 
I also tried to use the F3 key for the same effect to no avail. I have reinstalled and Uninstalled several times but the app refuses to let mission control be bound to any key..

The daemon is permitted to input monitoring and I have also unplugged and replugged all peripherals.

Attached below is my report generated just now.

Thanks for your work!

[ckb-next-report-1571271172.gz](https://github.com/ckb-next/ckb-next/files/3736882/ckb-next-report-1571271172.gz)

**Comment by NathanJava** [comment]
**[Fix This Issue:]**

Hey all, after you allow the next-daemon in S&P and this issue persists, be sure to check your Mission Control Prefs to verify that Catalina did not reset your bind for the application.

Below is a screenshot after I switched from Catalina default to a function key, which I then binded to the sniper button on my Harpoon to fix the issue. Hope this helps someone, anyone!


![Screen Shot 2019-10-22 at 5 51 20 PM](https://user-images.githubusercontent.com/56659954/67341166-bb1dca00-f4f4-11e9-94a9-ad7c5e1a606a.png)
